react/src/hooks/features/useFeatures.ts

```diff
@@ -23,6 +29,9 @@ export const useFeatures = ({
   if (assetTypes?.length) {
     endpoint += `?assetType=${assetTypes.join(',')}`;
   }
+  if (startDate && endDate && toggleDateFilter) {
+    endpoint += `&startDate=${startDate.toISOString()}&endDate=${endDate.toISOString()}`;
+  }
```

Comment:
On my staging maps, I'm not seeing the points on the map being filtered. Does start date and end date need to be as a query key [here? ](https://github.com/TACC-Cloud/hazmapper/blob/4216fa2ca37fc0039b3f6fe441b282fc299bce32/react/src/hooks/features/useFeatures.ts#L45)

```
  const query = useGet<FeatureCollection>({
    endpoint,
    key: [KEY_USE_FEATURES, { projectId, isPublicView, assetTypes }],
    options: { ...defaultQueryOptions, ...options },
  });
  return query;
};
```


https://github.com/user-attachments/assets/95e4cf85-275a-4727-9c8c-5ef82013a8a1